lib/node_modules/@stdlib/stats/base/dcumin/README.md

```diff
@@ -141,21 +141,15 @@ dcumin.ndarray( 4, x, 2, 1, y, -1, y.length-1 );
 <!-- eslint no-undef: "error" -->
 
 ```javascript
-var randu = require( '@stdlib/random/base/randu' );
-var round = require( '@stdlib/math/base/special/round' );
+var discreteUniform = require( '@stdlib/random/array/discrete-uniform' );
 var Float64Array = require( '@stdlib/array/float64' );
 var dcumin = require( '@stdlib/stats/base/dcumin' );
 
-var y;
-var x;
-var i;
-
-x = new Float64Array( 10 );
-y = new Float64Array( x.length );
-for ( i = 0; i < x.length; i++ ) {
-    x[ i ] = round( randu()*100.0 );
-}
+var x = discreteUniform( 10, -50, 50, {
+    'dtype': 'float64'
+});
 console.log( x );
+var y = new Float64Array( x.length );
```

Comment:
```suggestion

var y = new Float64Array( x.length );
```

Always include a line break after the last group of one or more consecutive `console.log` statements.